File: web3rpc/rpc-specs/code-samples/python/personal/unlockAccount.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"
passphrase = "helloWorld"
duration = 30

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.unlock_account(address, passphrase, duration)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/unlockAccount.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const address = "0x6b1ac7bda0073095df1d434c39d66fb31a592bdd";
  const passphrase = "helloWorld";
  const duration = 10;

  provider.personal
    .unlockAccount(
      address,
      passphrase,
      { duration },
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
